packages/amazonq/src/app/inline/completion.ts

```diff
@@ -43,19 +44,27 @@ export class InlineCompletionManager implements Disposable {
     private sessionManager: SessionManager
     private recommendationService: RecommendationService
     private lineTracker: LineTracker
-    private activeStateTracker: InlineGeneratingMessage
+    private incomingGeneratingMessage: InlineGeneratingMessage
+    private lineAnnotationTracker: LineAnnotationController
```

Comment:
nit: rename this to something like InlineTutorialAnnotation? I think naming this closer to the feature of how a user would describe it